app/Jobs/BroadcastNotification.php

```diff
@@ -34,12 +34,31 @@ class BroadcastNotification implements ShouldQueue
     private $receiverIds;
     private $source;
 
-    private static function beatmapsetReceiverIds($beatmapset)
+    private static function beatmapsetWatcherUserIds($beatmapset)
     {
-        return $beatmapset
-            ->watches()
-            ->pluck('user_id')
-            ->all();
+        return static::filterBeatmapsetModdingUserIds(
```

Comment:
I wonder if it's better to have a method in notification object itself (or its own class) to get all the watching users, both global and per-item

Reply:
probably move this in a separate PR